storage-bigtable/src/lib.rs

```diff
@@ -731,7 +737,7 @@ impl LedgerStorage {
             }
 
             self.connection
-                .delete_rows_with_retry("blocks", &[slot.to_string()])
```

Comment:
unless i'm reading code wrongly, this should be inconsistent with https://github.com/solana-labs/solana/pull/20044/files#r712516629?

Reply:
Correct, thanks